app/src/main/java/org/mozilla/focus/notification/NotificationUtil.java

```diff
@@ -30,7 +30,9 @@
                 .setColor(ContextCompat.getColor(context, R.color.surveyNotificationAccent))
                 .setAutoCancel(true)
                 .setContentIntent(pendingIntent)
-                .setPriority(NotificationCompat.PRIORITY_HIGH);
+                .setPriority(NotificationCompat.PRIORITY_HIGH)
+                // without setDefaults, the notification will not popup nor overlay
+                .setDefaults(NotificationCompat.DEFAULT_VIBRATE);
```

Comment:
While I know we expect heads-up. Is it expected to vibrate?

Reply:
Done